pkg/server/plugin/upstreamca/disk/disk.go

```diff
@@ -137,3 +123,38 @@ func New() (m upstreamca.Plugin) {
 		serialNumber: x509util.NewSerialNumber(),
 	}
 }
+
+func parsePrivateKey(keyFilePath string) (crypto.PrivateKey, error) {
```

Comment:
There is already shared code for this: see https://github.com/spiffe/spire/blob/master/pkg/common/pemutil/keys.go#L20